.cirrus.yml

```diff
@@ -28,6 +28,11 @@ fedora_35_task:
     - test -z $AVOCADO_SRC || python3 -m pip install $AVOCADO_SRC
     - python3 $SETUP
   bootstrap_script: &bootstrap_scr
+    # Pin the versions of tp-qemu and tp-libvirt to known working versions.
+    # These versions can be bumped at any time, but provide better understanding
+    # of regressions or version update requirements on the test providers side.
+    - python3 -c "from virttest.asset import ConfigLoader; cl = ConfigLoader('virttest/test-providers.d/io-github-autotest-qemu.ini'); cl.set('provider', 'ref', '33ed9b0ac6e4255877ff109a1d91fa84c9097dc5'); cl.save()"
+    - python3 -c "from virttest.asset import ConfigLoader; cl = ConfigLoader('virttest/test-providers.d/io-github-autotest-libvirt.ini'); cl.set('provider', 'ref', 'e6af7700f879fe1547bfb1421439121d35c919bd'); cl.save()"
```

Comment:
I agree with you that it is important to pin the versions of tp-qemu and tp-libvirt, so we will be changing only one component at the time.  But IMO we still need to update these version once in a while, and I am missing some well-defined rules how this will be done. Something like release check that, the tp-qemu and tp-libvirt were updated to the newest version in every release.

Reply:
The exact release steps need to be better ironed out indeed on the Avocado-VT side.  I can follow this up with a documentation change proposal, or open a discussion about it.

But I think both would take a lot more time (because of the discussion) than what is needed for a release related change.

Let me know if you are thinking of something simpler that can be addressed here.

Reply:
No, I agree with you that this needs to be well discuss, so I am ok with doing this after the release. Can you please open an issue or discussion about it, so we don't forget about this? Thanks

Reply:
Sure.  The issue tracking this is https://github.com/avocado-framework/avocado-vt/issues/3462 .